lk2nd/device/dts/msm8916/msm8916-qrd-9.dts

```diff
@@ -1,11 +1,22 @@
 // SPDX-License-Identifier: GPL-2.0-only
 
+#include <dt-bindings/gpio.h>
 #include <skeleton64.dtsi>
 #include <lk2nd.dtsi>
 
 / {
 	qcom,msm-id = <QCOM_ID_MSM8916 0>;
 	qcom,board-id = <QCOM_BOARD_ID_QRD 9>;
+
+	tlmm: tlmm {
+		compatible = "lk2nd,gpio";
+		phandle = <GPIO_DEV_PHANDLE(GPIOL_DEVICE_TLMM)>;
+	};
+
+	pm8916_pon: pm8916-pon {
```

Comment:
With pm8916-pon -> pmic-pon, can you try putting this into `lk2nd.dtsi` with `/omit-if-no-ref/`? Bonus points if you can check if it's really getting omitted if unused :)